manifests/a/Atlassian/Sourcetree/3.4.5/Atlassian.Sourcetree.installer.yaml

```diff
@@ -0,0 +1,16 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Atlassian.Sourcetree
+PackageVersion: 3.4.5

```

Comment:
```suggestion
PackageVersion: 3.4.5.0
```
You will need to edit the folder name as well from `manifests/a/Atlassian/Sourcetree/3.4.5/` to `manifests/a/Atlassian/Sourcetree/3.4.5.0/`
![image](https://user-images.githubusercontent.com/71691514/122600643-b6286980-d08d-11eb-84c5-b2509da2aa36.png)